pylj/pairwise.py

```diff
@@ -171,54 +187,15 @@ def lennard_jones_force(A, B, dr):
     float:
         The force between the two particles.
     """
-    print(
+    xacceleration(
```

Comment:
`xacceleration`?

Reply:
mistakenly changed, put back to print now